administrator/components/com_media/src/Model/ApiModel.php

```diff
@@ -405,6 +407,13 @@ public function delete($adapter, $path)
 			throw new \Exception($object->getError());
 		}
 
+		// Remove responsive versions if file is an image
```

Comment:
Just remove unused code, do not comment it in.

Reply:
Done